src/js/api/authFetch.mjs

```
@@ -15,5 +15,17 @@ export async function authFetch(url, options = {}) {
 	return fetch(url, {
 		...options,
 		headers: headers(),
-	});
+	})
+		.then((response) => {
+			if (!response.ok) {
+				return response.json().then((err) => {
+					throw new Error(JSON.stringify(err.errors, null, 2));
+				});
+			}
+
+			return response;
+		})
+		.catch((error) => {
+			throw error;
```

`catch block`: to suppress an error
`throw`: to create an error

Understand the differences, they have different purposes. Here you're basically catching an error and throwing it again.

**Suggestion:** remove the throw from the catch block and use the try/catch block in a UI/UX code block e.g event listener.